### PR TITLE
Added sponsorship button

### LINKED ISSUE #286: [FEATURE] add sponsorship button
### Description

add buymeacoffee button in the app and readme file. 

account to be used: https://www.buymeacoffee.com/

### PR DESCRIPTION
**Pull request by Deepu178**
### Feature
Closes #286 

### Changes proposed

Added Sponsorship Button 
"Buy Me A Coffee"
in README.md and beside "Submit Project"

### CODE REVIEW

**Comment by Deepu178** [comment]
@rohitdasu 
Please review my changes for "Sponsorship button"

**Comment by rohitdasu** [comment]
hi @Deepu178 thanks for the pr. 
two small changes needed. could you please make the buymeacoffe button height same as producthunt button height in topbar? also in mobile screen sidenavbar make it similar width. 
let me know if you have any questions. thanks

**Comment by Deepu178** [comment]
sure, @rohitdasu 
Thanks for feedback :)

**Comment by Deepu178** [comment]
Made changes as you suggested
looking forward for more issues 
@rohitdasu

File: components/Topbar/Topbar.tsx [hunk 1]
```diff
@@ -32,6 +32,18 @@ export const Topbar = () => {
...
+            target="_blank"
+            rel="noreferrer"
+            className="hidden md:block"
+          >
+            <img
+              src={`https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png`}
+              alt="Buy Me A Coffee"
+              style={{ height: '54px' }}
+            />
+          </a>
```
Comment (rohitdasu) [inline components/Topbar/Topbar.tsx:46]:
```suggestion
          {router.pathname !== SUBMIT_PAGE && router.pathname !== '/' && 
( <a
            href="https://www.buymeacoffee.com/rohit.dasu"
            target="_blank"
            rel="noreferrer"
            className="hidden md:block"
          >
            <img
              src={`https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png`}
              alt="Buy Me A Coffee"
              style={{ height: '54px' }}
            />
          </a> )}
```

Comment (rohitdasu) [reply components/Topbar/Topbar.tsx:46]: @Deepu178

Comment (Deepu178) [reply components/Topbar/Topbar.tsx:46]: @rohitdasu sure

**Review by rohitdasu — APPROVED** [review]

**Pull request merged**